### ISSUE TITLE
Please add native Wii Classic Controller Pro support

### ISSUE THREAD

**Issue by kevincrans** [cemu-project/Cemu#200]
For all versions at least up to 2.0-1
If I connect a wii-remote using Dolphin like always, everything works perfectly on all versions now.
However, while the nunchuck extension gets recognized, the classic controller makes the extension field stay None.

This is my favourite deprecated controller, so I really love to see it work one time.

There are no IR pointer and motion mappings, meaning the input settings window should grow anyways.

I recommend a radio button to swap the nunchuck mapping fields with the classic controller instead of a checkbox to remain compact.

Do you think I need a log file?

**Comment by kevincrans** [comment]
I have to add, I don't know what an U classic controller is, but if it is a misname I don't assume, this is about the physical controller, not just the mapping API.

**Comment by Squall-Leonhart** [comment]
the classic control is supported by SDL2

**Comment by kevincrans** [comment]
That's not fair, that the nunchuck cán connect without SDL, while the classic controller can't.

Cemu should support the whole wii-remote with classic controller without needing to install tools seperately, no matter how, just like Dolphin.

It's so wastefull, since it's as easy to add as the nunchuck.

**Comment by Squall-Leonhart** [comment]
the wiiu games that can use the classic control can be counted on a single hand, these already work via the wiimote api.

Usage with anything else requires mapping the control to the wii u input devices, which SDL2 provides for, nothing further is required from Cemu's end.

**Comment by kevincrans** [comment]
Not true, the cemu wiimote api is incomplete, else the classic controller would show up and pressing the buttons would map/do anything.

**Comment by Squall-Leonhart** [comment]
Can be closed with the merging of https://github.com/cemu-project/Cemu/pull/1851

**Comment by Squall-Leonhart** [comment]
> the cemu wiimote api is incomplete

not exactly, the classic controls are implemented, via the "Wii U Classic Controller" option, you can then map a classic controller extension via the wii mote api.

There was a bug in the logic that prevented the buttons from all assigning properly, this will be fixed with https://github.com/cemu-project/Cemu/pull/1851
